providers/implementations/kem/ec_kem.c

```diff
@@ -630,17 +602,16 @@ static int derive_secret(PROV_EC_CTX *ctx, unsigned char *secret,
         goto err;
 
     /* kemctx is the concat of both sides encoded public key */
-    memcpy(kemctx, sender_pub, ctx->alg->encodedpublen);
-    memcpy(kemctx + ctx->alg->encodedpublen, recipient_pub,
-           ctx->alg->encodedpublen);
+    memcpy(kemctx, sender_pub, info->Npk);
```

Comment:
OPENSSL_memcpy

Reply:
does ``OPENSSL_memcpy()`` exist? not seeing it here
```
$ find . -name '*.h' -exec grep -l OPENSSL_memcpy {} \;
$
```

Reply:
Yeah, there is no such thing.

Reply:
Sorry, I made a memory confusion with `OPENSSL_memdup`. there no need to a OPENSSL_memcpy.